joe/joe.py

```diff
@@ -40,18 +41,18 @@ def _get_data_dir(path):
 
 def _walk_gitignores():
     '''Recurse over the data directory and return all .gitignore file names'''
-    l = []
-    for root, subFolders, files in os.walk(DATA_DIR):
-        l += [f.replace('.gitignore', '') \
-                for f in files if f.endswith('.gitignore')]
-    return sorted(l)
+    gitignore = []
+    for _root, _sub_folders, files in os.walk(DATA_DIR):
+        gitignore += [f.replace('.gitignore', '')
+                      for f in files if f.endswith('.gitignore')]
```

Comment:
Because that line verbatim says `for f in files`, it's simpler to leave it at f in this case.